core/src/main/scala/caliban/introspection/adt/__InputValue.scala

```diff
@@ -13,6 +13,7 @@ case class __InputValue(
   description: Option[String],
   `type`: () => __Type,
   defaultValue: Option[String],
+  isOptional: Boolean = false,
```

Comment:
This class must adhere to the spec: https://spec.graphql.org/October2021/#sec-The-__InputValue-Type
The optionality is already expressed by `type` (see https://spec.graphql.org/October2021/#sec-The-__Type-Type.Non-Null).